Trash

jamsic opened the pull request.

KorixD commented on BayesModule/src/main/java/bayes/Bayes.java:
```diff
@@ -0,0 +1,317 @@
+
+    public static int getnGrammCountFromDB(String nGramm, int postId) {
+        if (postId == 0) {
+            if (nGramm.equals("зверь")) {
```
Мы же это уже переделывали на более простую структуру данных?

KorixD commented on BayesModule/src/main/java/bayes/Bayes.java:
```diff
@@ -0,0 +1,317 @@
+        Classifier cModel = (Classifier)new NaiveBayesMultinomial();
+        try {
+            cModel.buildClassifier(isTrainingSet);
+        } catch (Exception e) {
```
Ловить Exception не очень хорошо, потому что мы по сути хотим поймать и обработать одно конкретное исключение, а не все (что делать со всеми, в которые, например, входит NPE, обычно не понятно).

sssmaximusss replied:
Метод buildClassifier(...) выкидывает именно Exception.
В его реализации есть явное throw Exception(...).
Тут по-другому никак.

KorixD commented on BayesModule/src/main/java/bayes/Bayes.java:
```diff
@@ -0,0 +1,317 @@
+        double[][] cmMatrix = eTest.confusionMatrix();
+
+
+        //FastVector fvWekaAttributes2 = new FastVector(3);
```
убрать все закомментированное

KorixD commented on BayesModule/src/main/java/bayes/BeijingExampleMultinomialNaiveBayes.java:
```diff
@@ -0,0 +1,254 @@
+
+public class BeijingExampleMultinomialNaiveBayes {
+
+    public static Instance getNextInstance(FastVector fvWekaAttributes, int[] occurrencescount, String label) {
```
occurrencesСount
И, заодно, сделать автоматическое форматирование на всём коде.

KorixD commented on BayesModule/src/main/java/bayes/BeijingExampleMultinomialNaiveBayes.java:
```diff
@@ -0,0 +1,254 @@
+
+
+        {
+            // Chinese Beijing Chinese
```
Лучше в метод

KorixD commented on Crawler/src/main/java/crawler/Crawler.java:
```diff
@@ -13,18 +13,16 @@
 import org.apache.http.HttpHost;
```
Почему у тебя отличается этот файл? Стоит сделать мердж с мастером?

KorixD commented on Crawler/src/main/java/crawler/proxy/ProxyFactory.java:
```diff
@@ -1,175 +1,26 @@
 package crawler.proxy;
```
Этот файл тоже не относится к твоему коду.
И те, что ниже

KorixD commented on dataset/src/main/java/dataset/Dataset.java:
```diff
@@ -0,0 +1,274 @@
+
+    private void setnGramAttributeIndex(List<String> ngramsList) {
+        int i = 0;
+        this.totalnGramsListIndexes = new HashMap<String, Integer>();
```
можно без this.

KorixD commented on dataset/src/main/java/dataset/Dataset.java:
```diff
@@ -0,0 +1,274 @@
+    public static String tag1 = "детское";
+    public static String tag2 = "статистика";
+
+    private AttributeSelection selecter;
```
selector

KorixD commented on dataset/src/main/java/dataset/LSA.java:
```diff
@@ -0,0 +1,793 @@
+    static final long serialVersionUID = -8712112988018106198L;
+
+    /** The data to transform analyse/transform */
+    private Instances m_trainInstances;
```
Переведи код этого класса, в соответствие с java code conventions, пожалуйста. Это просто сделать, например, с помощью рефакторинга из окна IDE.

Allight7 committed:
added method for getting most popular in normalized posts tags from DB. also test example to it.

# Conflicts:
#	db/src/main/java/db/DBConnector.java
#	db/src/test/java/DBConnectorTestLearning.java

KorixD commented on BayesModule/src/main/java/bayes/BayesFromDB.java:
```diff
@@ -0,0 +1,777 @@
+        return isTrainingSet;
+    };
+
+    /*
```
все равно много закомментированного кода. он действительно нужен?

KorixD commented on Crawler/src/main/java/Main.java:
```diff
@@ -1,21 +0,0 @@
-import crawler.Crawler;
```
Сделай git checkout master -- Crawler/src/main/java/Main.java (забери их из ветки мастер, чтобы не показывались измененными здесь) и точно так же со всеми файлами, которые не относятся к пр'у.

KorixD replied:
Или git checkout <хеш коммита до твоих изменений> -- <название файла>

sssmaximusss committed:
Merge remote-tracking branch 'origin/trash' into trash

sssmaximusss committed:
small refactoring code for pr

sssmaximusss committed:
solving conflict by add removing module

sssmaximusss committed:
extra adding files for solving conflicts

sssmaximusss committed:
and last adding
